cluster/nats_rpc_server_test.go

```diff
@@ -335,7 +335,7 @@ func TestNatsRPCServerInitShouldFailIfConnFails(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	mockSessionPool := sessionmocks.NewMockSessionPool(ctrl)
 	rpcServer, _ := NewNatsRPCServer(*cfg, sv, nil, nil, mockSessionPool)
-	mockSessionPool.EXPECT().OnSessionBind(rpcServer.onSessionBind)
+	//mockSessionPool.EXPECT().OnSessionBind(rpcServer.onSessionBind)
```

Comment:
do you know why this is not happening anymore?

Reply:
I think this was an assertion bug on older mock, this is a duplicate expect, you'll see the other one a few lines below.